src/pages/my-lessons/MyLessons.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+import Box from '@mui/material/Box'
+
+import Tab from '~/components/tab/Tab'
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+
+import { listDataMock } from '~/pages/my-lessons/MyLessons.constants'
+
+import { styles } from '~/pages/my-lessons/MyLessons.styles'
+
+const MyLessons = () => {
+  const [value, setValue] = useState<number>(0)
+
+  const handleClick = (newValue: number) => {
+    setValue(newValue)
+  }
+
+  const tabs = listDataMock.map((item) => (
+    <Tab
+      activeTab={value === item.id}
+      key={item.id}
+      onClick={() => handleClick(item.id)}
+    >
+      {item.title}
+    </Tab>
+  ))
+
+  const itemsToDisplay = listDataMock[value].items.map((item) => (
+    <Tab activeTab={false} key={item.id} onClick={() => null}>
+      {item.title}
+    </Tab>
+  ))
+
+  return (
+    <PageWrapper>
+      <Box sx={styles.wrapper}>
```

Comment:
its redundant , give needed styles to pageWrapper

Reply:
PageWrapper has Container with style flex-column, so it will ignore my styles in my component MyLessons. That's why i give styles to Box

Reply:
it wont , just pass flex direction row

Reply:
thanks, I forget about it😅. fixed